presto-main/src/main/java/com/facebook/presto/server/CoordinatorModule.java

```diff
@@ -189,6 +189,7 @@ protected void setup(Binder binder)
 
         // dispatcher
         binder.bind(DispatchManager.class).in(Scopes.SINGLETON);
+        newExporter(binder).export(DispatchManager.class).withGeneratedName();
```

Comment:
For now, I have exposed the DispatchManager QueryManagerStats in a separate namespace. But the stats object exposed via this class has more accurate stats(compared to SqlQueryManager) for all metrics including Queued Query and Running Query count. 

If we decide to expose only one of them, I would recommend we should expose the DispatchManager stats object. To make the change backwards compatible we can expose it under the same namespace as before so it remains transparent to the users.